config/webpack.config.js

```diff
@@ -121,7 +121,10 @@ module.exports = {
 		outputStyle: 'expanded'
 	},
 	postcss: () => {
-		return [ autoprefixer({ browsers: ['> 1%', 'last 2 versions', 'ie >= 8', 'ff ESR', 'bb >= 7', 'iOS >= 5'] }) ];
+		return [ autoprefixer({
+			browsers: ['> 1%', 'last 2 versions', 'ie >= 8', 'ff ESR', 'bb >= 7', 'iOS >= 5'],
```

Comment:
I'd rather leave that to somebody else to implement and QA. For the moment I'm just applying a better implementation of something we've been trying to do hackily for ages